membership_policy.md

```diff
@@ -0,0 +1,125 @@
+| Reviewer | Review contributions from others | * Member<br>* History of review and authorship in the project<br> * Sponsored by an approver | [OWNERS](https://github.com/kubevirt/kubevirt/blob/master/OWNERS_ALIASES) file reviewer entry |
+| Approver| Approve accepting contributions | * Reviewer<br>* Highly experienced<br> * Active reviewer & contributor to the project |  Sponsored by 2 approvers| [OWNERS](https://github.com/kubevirt/kubevirt/blob/master/OWNERS_ALIASES) file approver entry |
+
+
+## New contributors
+
+[New contributors](https://github.com/kubevirt/kubevirt/blob/master/CONTRIBUTING.md) should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
+All contributors are expected to adhere to the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the principles in this document, familiarity with project organization, roles, policies, procedures, conventions, etc., and technical and/or writing ability. Role-specific expectations, responsibilities, and requirements are enumerated below.
+
+## Member
+
+Members are active contributors in the community. They can have issues and PRs assigned to them, participate in community related meetings (e.g. bug scrub), and pre-submit tests are automatically run for their PRs. Members are expected to remain active contributors to the community.
+Defined by: Member of the KubeVirt GitHub organization
+
+### Requirements
+
+  * Enabled [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication) on their GitHub account
+  * Actively contributing to the project with multiple contributions to the project or community. Contribution may include, but is not limited to:
+    * Authoring or reviewing PRs on GitHub
+    * Filing or commenting on issues on GitHub
+    * Contributing to community discussions (e.g. meetings, Slack, email discussion forums, Stack Overflow)
+    * GitHub Insight metrics will be reviewed periodically to
+  * Subscribed to kubevirt-dev@googlegroups.com
+  * Has read the [contributor guide](CONTRIBUTING.md)
+  * Actively contributing to the project
+  * Sponsored by 2 reviewers. Note the following requirements for sponsors:
+    * Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating on issues, etc.
+    * Sponsors must be reviewers or approvers in the KubeVirt OWNERS file.
+    * Sponsors should be from multiple member companies to demonstrate integration across community.
+  * [Open an issue](https://github.com/kubevirt/kubevirt/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E) against the KubeVirt/org repo
```

Comment:
I think @rmohr has a point above, maybe we can rephrase this into "Open a PR against the [org members section](https://github.com/kubevirt/project-infra/blob/master/github/ci/prow/files/orgs.yaml#L21)" ?